src/SlackConnector/Connections/Clients/Chat/FlurlChatClient.cs

```diff
@@ -12,18 +12,19 @@ namespace SlackConnector.Connections.Clients.Chat
     internal class FlurlChatClient : IChatClient
     {
         private readonly IResponseVerifier _responseVerifier;
-        internal const string SEND_MESSAGE_PATH = "/api/chat.postMessage";
+        internal const string POST_MESSAGE_PATH = "/api/chat.postMessage";
```

Comment:
sorry about the rename, when I added "chat.update" it seemed that would be nice to have the variables coincide with underlying path name as well and renamed send message to post message...